files/uk/glossary/accessible_name/index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Доступна назва
+slug: Glossary/Accessible_name
+page-type: glossary-definition
+---
+
+{{GlossarySidebar}}
+
+**Доступна назва** – це назва елемента користувацького інтерфейсу; це текст, пов'язаний з елементом HTML, який служить користувачам допоміжних технологій підписом елемента.
+
+Доступні назви доносять призначення чи задум елемента. Це допомагає користувачам зрозуміти, для чого цей елемент і як з ним взаємодіяти. Загалом, доступні назви елементів повинні бути неповторним на всій сторінці. Це допомагає користувачам відрізняти елемент від інших елементів і визначити елемент, з яким хочуть взаємодіяти.
+
+Залежно від елемента та розмітки HTML, значення доступної назви може виводитися від видимого (наприклад, тексту всередині {{HTMLElement("figcaption")}}) або невидимого (наприклад, атрибута `aria-label`, заданого на елементі) вмісту, або ж від поєднання першого та другого. Те, як визначається доступна назва елемента, засновано на [обчисленні доступної назви](https://www.w3.org/WAI/ARIA/apg/practices/names-and-descriptions/#name_calculation), яке відрізняється для різних елементів.
```

Comment:
```suggestion
Залежно від елемента та розмітки HTML, значення доступної назви може виводитися від видимого (наприклад, тексту всередині {{HTMLElement("figcaption")}}) або невидимого (наприклад, атрибута `aria-label`, заданого на елементі) вмісту, або ж від поєднання першого та другого. Те, як визначається доступна назва елемента, ґрунтується на [обчисленні доступної назви](https://www.w3.org/WAI/ARIA/apg/practices/names-and-descriptions/#name_calculation), яке відрізняється для різних елементів.
```
https://onlinecorrector.com.ua/uk/%D1%89%D0%BE-%D2%91%D1%80%D1%83%D0%BD%D1%82%D1%83%D1%94%D1%82%D1%8C%D1%81%D1%8F-%D0%BD%D0%B0/